src/components/DateRangePicker.jsx

```diff
@@ -281,6 +282,17 @@ class DateRangePicker extends React.PureComponent {
     this.container = ref;
   }
 
+  closeDayPicker() {
+    const { onFocusChange } = this.props;
+    this.setState({
```

Comment:
If the keyboard shortcut screen was open, I'd expect hitting escape would close it. If the calendar was open, I'd expect hitting escape would close it. But, if the calendar AND the keyboard shortcut screen were open, I'd expecting hitting escape would only close one (the one on top, which is the keyboard shortcut screen in this case).